cli/clear.go

```diff
@@ -0,0 +1,27 @@
+package cli
+
+import (
+	"gopkg.in/alecthomas/kingpin.v2"
+)
+
+func ConfigureClearCommand(app *kingpin.Application, a *AwsVault) {
+	input := RemoveCommandInput{}
+
+	cmd := app.Command("clear", "Removes sessions for the profile")
+
+	cmd.Arg("profile", "Name of the profile").
+		Required().
+		HintAction(a.MustGetProfileNames).
+		StringVar(&input.ProfileName)
```

Comment:
I'm not sure we should require a profile here. Maybe something like `aws-vault clear-cache` should clear all cached sessions or credentials for all profiles. (Note that the recent SSO addition means that there may be other types of cached credentials to clear, still to be implemented in #571). Thoughts?